bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -1196,6 +1196,23 @@
 			<Label lang="en">Z-Wave Thermostat</Label>
 			<ConfigFile>rtc/ct100.xml</ConfigFile>
 		</Product>
+		<Product>
```

Comment:
You are probably right, may be errors in the pepper db. The pepper db shows 2 versions (http://www.pepper1.net/zwavedb/device/701 and http://www.pepper1.net/zwavedb/device/764) for the US and EU radio versions.  Those are the numbers I used.  I've only tested one, the US version but it matches the EU product id..
However, every now and again mine shows up as id 100, type 002 for some reason during initialisation which is why that was added - but that may be another story.